packages/orchestration/src/examples/stakeBld.contract.js

```diff
@@ -75,6 +95,22 @@ export const start = async (zcf, privateArgs, baggage) => {
         }),
       );
     },
+    async makeAccount() {
+      trace('makeAccount');
+      return makeLocalAccount().then(({ holder }) => holder);
+    },
```

Comment:
use `.then` or `async`. for the latter,
```suggestion
    async makeAccount() {
      trace('makeAccount');
      const { holder } = makeLocalAccount();
      return holder;
    },
```

Reply:
Thanks, oversight